devops/excludeprojects.txt

```diff
@@ -1,2 +1,4 @@
 .sln
-.csproj
\ No newline at end of file
+.csproj
```

Comment:
It should be better to use another name - excludelist.txt | exclude.txt | excludeitems.txt | excludelist.txt

Reply:
Renamed both: excludefiles.txt -> exclude-common.txt; excludeprojects.txt -> exclude-for-model.txt